SystemServices/CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ All notable changes to this RDK Service will be documented in this file.
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
 
+## [1.0.8] - 2022-12-05
+### updated the system json file
+- RDK-39389: Add Doc for onTerritoryChanged & onTimeZoneDSTChanged events to System Plugin
```

Comment:
We don't need changelog entry. Also fyi don't mention JIRA tickets. Pls refer to README above in the CHANGELOG.